packages/dd-trace/test/appsec/iast/analyzers/index.spec.js

```diff
@@ -0,0 +1,43 @@
+'use strict'
+
+const proxyquire = require('proxyquire')
+const Analyzer = require('../../../../src/appsec/iast/analyzers/vulnerability-analyzer')
+
+describe('Analyzers index', () => {
+  let analyzers
+  let analyzerA
+  let analyzerB
+  let fakeAnalyzers
+
+  beforeEach(() => {
+    analyzerA = new Analyzer()
+    analyzerB = new Analyzer()
+
+    fakeAnalyzers = {
+      analyzerA: analyzerA,
+      analyzerB: analyzerB
+    }
+
+    Analyzer.prototype.configure = sinon.spy()
+
+    analyzers = proxyquire('../../../../src/appsec/iast/analyzers', {
+      './analyzers': fakeAnalyzers
+    })
+  })
+
+  afterEach(() => {
+    sinon.restore()
+  })
+
+  it('should enable all analyzers', () => {
+    analyzers.enableAllAnalyzers()
+    expect(fakeAnalyzers.analyzerA.configure).to.have.been.calledWith(true)
```

Comment:
Maybe is just me... but I like when the test are concrete, for example, I like when the method calledOnce is used instead of called.
Does exist `calledOnceWith` method? At this way we are sure that we are not calling twice